draft-barnes-sframe-mls.md

```diff
@@ -57,17 +87,25 @@ sender_base_key[index] = HKDF-Expand(sframe_epoch_secret,
                            encode_big_endian(index, 4), AEAD.Nk)
 ~~~~~
 
-For compactness, do not send the whole epoch number.  Instead, we send only its
-low-order E bits.  Note that E effectively defines a re-ordering window, since
-no more than 2^E epoch can be active at a given time.  Receivers MUST be
-prepared for the epoch counter to roll over, removing an old epoch when a new
-epoch with the same E lower bits is introduced.  (Sender indices cannot be
-similarly compressed.)
+The Key ID (KID) field in the SFrame header provides the epoch and index values
+that are needed to generate the appropriate key from the MLS key schedule.
```

Comment:
The alternative might be to use the SecretTree from MLS which provides more fine-grained FS guarantees. But it might be overkill for real-time applications.

Reply:
Added an open issue.

Reply:
Minor point: The issue is not with this KID construction, since the KID would have this form even if we use the secret tree.